aries-site/src/themes/aries.js

```diff
@@ -240,6 +240,8 @@ export const aries = deepMerge(hpe, {
     overlay: {
       background: '#0000001F',
     },
+    // temp CSS selector to target Layer overlay
+    extend: '> div { backdrop-filter: blur(12px); }',
```

Comment:
Should we align this blur with the same that we have on `datatable.pinned`? What is the visual difference between 8px vs 12px? While we are looking at blur and with the Firefox support, are we able to clean up the opacity in `datatable.pinned` as well?

Reply:
Here is 12px vs 8px. Personally, I prefer the extra blurring on 12px because it softens it to the point where my eyes aren't trying to make out the content at all. 8px feels a little distracting to me.

12px
<img width="1577" alt="Screen Shot 2023-01-27 at 12 59 25 PM" src="https://user-images.githubusercontent.com/12522275/215200562-4b068e97-fc5c-4e93-a12e-d0cdf9966800.png">

8px
<img width="1579" alt="Screen Shot 2023-01-27 at 1 10 25 PM" src="https://user-images.githubusercontent.com/12522275/215200586-1017a3f7-ee13-49ec-a076-c10467d04ef6.png">

Agree that will Firefox support of backdrop-filter now, we can cleanup datatable.pinned. I'll look at that.

Reply:
On DataTable, the difference between at 8px and 12px blur isn't really noticeable so I'd pitch that whatever we decide for layer we update DataTable.pinned to follow.

Reply:
right on. Let's go with 12px for now. We can refine when we get into true support for all of these changes.